modules/org.kie.kogito.rhpam.builder/module.yaml

```diff
@@ -1,5 +1,5 @@
 name: org.kie.kogito.rhpam.builder
-version: "7.11.0"
+version: "7.13.0"
```

Comment:
I believe here it should be 7.13.1? The rest of the PR looks good.

Reply:
does not seem to be replaced ...